build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -0,0 +1,230 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using Microsoft.DotNet.Cli.Build.Framework;
+using Microsoft.DotNet.InternalAbstractions;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using static Microsoft.DotNet.Cli.Build.Framework.BuildHelpers;
+using static Microsoft.DotNet.Cli.Build.FS;
+
+namespace Microsoft.DotNet.Cli.Build
+{
+    public class CompileTargets
+    {
+        public static readonly string CoreCLRVersion = "1.0.2-rc2-24027";
```

Comment:
This file and string is updated in the `update-dependencies` script.  Since you are moving the `dotnet-cli-build\CompileTargets.cs` and duping it here, can you update the `update-dependencies` script for the new location, and add this file as well?